source/feature-guide/feature-analyzing.rst

```diff
@@ -653,3 +654,78 @@ against another variable "449b421":
 
 The result will be an array of output cubes:
 
+.. language_specific::
+   --JSON
+   .. code:: json
+
+      {
```

Comment:
@jonkeane Is this the format that's going to be returned everywhere? Currently in `whaam`, we have multiple cubes, and not a single JSON (that contains more cubes). The reason I'm asking is, obviously, `cr.exporter`, but I also want to be sure what this implicates...